README.md

```diff
@@ -594,7 +594,10 @@ The mountPath should be changed to `/var/run/cri.sock` and hostPath should be po
 `/var/run/containerd/containerd.sock` for containerd. If using helm chart, the flag `--set cri.hostPath.path=/var/run/containerd/containerd.sock`
 can set the paths for you.
 
-*Note*: When using other container runtime instead of dockershim, make sure also setting kubelet in instances.
+*Note*: 
+
+* When using other container runtime instead of dockershim, make sure also setting kubelet in instances.
```

Comment:
"make sure also setting kubelet in instances." - this seems to be incomplete.

Reply:
Yeah, agree on the above. Maybe something along the lines of - `When using a different container runtime instead of dockershim in VPC CNI, make sure `kubelet` is also configured to use the same CRI` and we can leave a link to `containerd` configuration flow - https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html#containerd-bootstrap